GliaWidgets/Sources/Theme/Theme.swift

```diff
@@ -61,6 +61,12 @@ public class Theme {
     /// Controls the visibility of the "Powered by" text and image.
     public var showsPoweredBy: Bool
 
+    /// Snack bar View style.
+    public lazy var snackBar: SnackBarStyle = { return snackBarStyle }()
```

Comment:
I am thinking about the snackBar and invertedSnackBar. I think that for now, snackBars can only be triggered by us, correct? However for example the alerts are also only triggered by us and each one has their own alert style. So do you think that it's fine if there's just a generic `snackBar` and `invertedSnackBar`, or you think that it would be better to have one for live observation to allow us to have future snackBars?

Reply:
> I think that for now, snackBars can only be triggered by us, correct? 
Correct, and snackback could appear on Glia UI or on integrator app UI.

> So do you think that it's fine if there's just a generic snackBar and invertedSnackBar, or you think that it would be better to have one for live observation to allow us to have future snackBars?
Perhaps I didn't get your question correctly, so I'll rephrase it - "Do we need to have usual and inverted values?"
The main problem is that during the call visualizer session snack bar will appear on top of client UI, so maybe the default snackBar style will not be acceptable for the integrator and they will need to override it. 
+ For now, SDK creates these styles based on default palette, but again integrator may change the theme and receive not nice snackbar, so I'd propose retaining these interfaces separately.

Reply:
What I meant was that all snackBars will have either `snackBar` or `invertedSnackBar`. But in alerts, all alerts have their own style. So following that logic, the CV snacbkar would be something like `cvSnackbarStyle`. And the one for engagement would be `engagementSnackbarStyle`. That way in the future if there's another type of snackbar, it would be `thirdSnackbarStyle` instead of using either `snackBar` or `invertedSnackBar`. My worry is that `snackBar` or `invertedSnackBar` are sufficient until they are not, and then we would need to deprecate them to have specifics for each snackBar. But it's impossible to predict the future, so I don't know exactly how to proceed.